### ISSUE TITLE
This repo. is VAE, but not a GMVAE?

### ISSUE THREAD

**Issue by shahuzi** [rishikksh20/gmvae_tacotron#1]
Hi, I've read your code, and it seems that your implementation is a VAE-Tacotron, but not a GMVAE-Tacotron?
Because the loss here seems is no difference from this [repo.](https://github.com/rishikksh20/vae_tacotron2)?

**Comment by rishikksh20** [comment]
@shahuzi yes because this way I am able to train a model easily. But in this repo I am sampling from Gaussian distribution.

**Comment by xiaoyangnihao** [comment]
> @shahuzi yes because this way I am able to train a model easily. But in this repo I am sampling from Gaussian distribution.

Can you make more explanations about this repo ? For example, the kl_loss between VAE-Tacotron and GMVAE-Tacotron is same ?

**Comment by zpcoftts** [comment]
Did u find some other GMVAE repo , which can follow?
